### DISCUSSION TITLE
[react native] support react native

### DISCUSSION THREAD

**Discussion by radiorz** [DouyinFE/semi-design#1366] [Q&A]
### Which Component Need Enhancement 期望新增功能的组件
- All

### Feature Description 功能描述
听说 semi design 通过 fundation 和 adapter 相分离, 可以适配多个框架.
这看起来很符合 headless ui 的概念.
而 headless ui 通常可以适配 react native.
那么请问 react native 支持么, 支持如何与如何使用.

**Comment by SudoUserReal** [comment]
关于 foundation/adapter 可以参考这篇文章。https://github.com/DouyinFE/semi-design/wiki/About-Foundation--Adapter

目前 foundation 部分是可以直接在 ReactNative 上用的，UI 部分需要用户自己写 <View/>， 因为官方只提供了 dom 版本的。 如果你只需要部分组件的话可以自己先写UI 然后复用 foundation，如果你是要适配出全部组件的 UI 层(即 adapter) ，做一个完整的组件库，我们会提供必要的帮助和咨询。

**Comment by SudoUserReal** [comment]
使用 foundation 直接安装 @douyinfe/semi-foundation 这个包就可以。
